docs/sphinx/manual/creating_hpx_projects.rst

```diff
@@ -419,7 +425,9 @@ run it on.
 
 .. literalinclude:: ../../examples/quickstart/hello_world_1.cpp
    :language: c++
-
+   :start-after://[hello_world_1_getting_started
+   :end-before://]
```

Comment:
```suggestion
   :start-after: //[hello_world_1_getting_started
   :end-before: //]
```

Reply:
And the same below, just for consistency with how we have it elsewhere.

Reply:
Got it!